crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -559,15 +560,33 @@ impl PgManager {
 impl TryFrom<StoredCheckpoint> for Checkpoint {
     type Error = Error;
     fn try_from(c: StoredCheckpoint) -> Result<Self, Self::Error> {
+        let checkpoint_commitments: Vec<CheckpointCommitment> =
+            bcs::from_bytes(&c.checkpoint_commitments).map_err(|e| {
+                Error::Internal(format!(
+                    "Can't convert checkpoint_commitments into CheckpointCommitments. Error: {e}",
+                ))
+            })?;
+
+        // TODO: validate this
+        let live_object_set_digest =
+            checkpoint_commitments
+                .last()
+                .map(|commitment| match commitment {
+                    CheckpointCommitment::ECMHLiveObjectSetDigest(digest) => {
+                        Digest::from_array(digest.digest.into_inner()).to_string()
+                    }
+                });
```

Comment:
only checking the last commitment seems inaccurate, especially if they're could be more later

Reply:
I don't really understand the Vec<CheckpointCommitment>, my assumption was that the last in the vector would be the most recent one

Reply:
cc @williampsmith for how this field is populated

Reply:
This does the right thing for now, as we only have a single commitment (vec of size 1) with a single enum variant (`ECMHLiveObjectSetSigest`). 

If you want to be future proof, it would be better to instead filter the vector for the variant, as even if we add other variants in the future, we'll only ever have one element of `CheckpointCommitment::ECMHLiveObjectSetDigest`. But either way in that future, this code would need to change so this is probably fine